client/src/pages/Home.js

```diff
@@ -27,19 +27,15 @@ export default function Home() {
         </div>
```

Comment:
Massive purge on the content here to simplify for now. 
> Need to cleanup/organize so it's clear this is a concept app and so that content matches design